data/templates/dhcp-server/kea-dhcp4.conf.j2

```diff
@@ -50,6 +50,19 @@
                 "space": "ubnt"
             }
         ],
+{% if dynamic_dns_update is vyos_defined %}
+        "dhcp-ddns": {
+            "enable-updates": true,
+            "server-ip": "127.0.0.1",
+            "server-port": 53001,
```

Comment:
I wonder if these ports should be configurable, in case someone is already running something on 53001.